src/App.tsx

```diff
@@ -48,34 +66,59 @@ export function getReorderedGoods(
 // };
 
 export const App: React.FC = () => {
+  const [isReversed, setReversed] = React.useState(false);
+  const [sortType, setSortType] = React.useState(SortType.NONE);
+
   return (
     <div className="section content">
       <div className="buttons">
-        <button type="button" className="button is-info is-light">
+        <button
+          type="button"
+          className={`button is-info ${sortType !== SortType.ALPHABET ? 'is-light' : ''}`}
```

Comment:
There is library in every react project that allows you to write conditional classNames. Take a look into that lib documentation for future reference. https://www.npmjs.com/package/classnames

`"classnames": "^2.5.1"`